src/segment-loader.js

```diff
@@ -15,6 +15,7 @@ import { minRebufferMaxBandwidthSelector } from './playlist-selectors';
 
 // in ms
 const CHECK_BUFFER_DELAY = 500;
+const ABORT_EARLY_BLACKLIST_MILLIS = 1000 * 60 * 2;
```

Comment:
I agree that if we had to abort early because of poor network, it is likely that it will still be an issue in the future. My concern is that this would happen during startup or very early in playback, but then as we have substantial forward buffer, we could up-switch without causing rebuffering, but it may still be within that 2 minutes. This could just be a hypothetical situation that wouldn't actually play out in